noxfile.py

```diff
@@ -1,7 +1,7 @@
 #
```

Comment:
REVIEWERS: This PR also removes the algol60 dependency in the workflows since this package is supposed to be independent of HPE resources.